src/serializer.ts

```diff
@@ -44,16 +47,29 @@ class Serializer {
       Fee: '',
     }
 
-    object.Sequence = transaction.getSequence()
-    object.SigningPubKey = Utils.toHex(transaction.getSigningPublicKey_asU8())
-    object.LastLedgerSequence = transaction.getLastLedgerSequence()
+    const sequence = transaction.getSequence()?.getValue()
+    if (sequence) {
+      object.Sequence = sequence
+    }
+
+    const signingPubKeyBytes = transaction
+      .getSigningPublicKey()
+      ?.getValue_asU8()
+    if (signingPubKeyBytes) {
+      object.SigningPubKey = Utils.toHex(signingPubKeyBytes)
+    }
+
+    const lastLedgerSequence = transaction.getLastLedgerSequence()?.getValue()
+    if (lastLedgerSequence) {
+      object.LastLedgerSequence = lastLedgerSequence
+    }
```

Comment:
Before the object always had a `LastLedgerSequence` property, and now it's conditional. Why is that?

Reply:
The old code was probably incorrect because the default value of protocol buffers numbers are 0.  We shouldn't require folks to set this optional property, so if it isn't set we omit if from the  JSON object. 

I'm applying this same logic to the above sequence and signing public key properties.